tuta-sdk/rust/src/lib.rs

```diff
@@ -122,6 +131,10 @@ impl Sdk {
     pub fn mail_facade(&self) -> MailFacade {
         MailFacade::new(self.entity_client.clone())
     }
+
+    pub fn user_facade(&self) -> UserFacade {
```

Comment:
I don't know if we need to expose it to the users, maybe not yet?

Reply:
I'm just using this as a way for testing unencrypted entities. Is there a better one than `User`?

Reply:
it's okay